fleaapi/controllers/homepage.py

```diff
@@ -0,0 +1,64 @@
+from django.http import JsonResponse
+# from .models import Item
+from fleaapi.models import Item
+
+# get homepage items API
+def get_homepage_items(request):
+    items = list(Item.objects.values())
+    return JsonResponse({'items': items})
+
+# get category items API
+def get_category_items(request):
```

Comment:
I don't think we need all these apis in this PR. In my opinion this PR should contain one API that returns a json that fills contents on the homepage. Please remove them from this PR to keep the changes small and clean